go.mod

```diff
@@ -38,6 +38,8 @@ require (
 	sigs.k8s.io/hierarchical-namespaces v0.9.0
 )
 
+require github.com/stretchr/testify v1.7.1 // indirect
```

Comment:
wondering why this is happening 🤔  it should have gone in the block below with all the other indirect dependencies, right?

Reply:
We noticed that, too. I think that `go mod tidy` added it, but I'm not sure why.